include/ListView/ListViewSubPanel.php

```diff
@@ -261,7 +261,7 @@ public function process_dynamic_listview_rows($data, $parent_data, $smartyTempla
             reset($data);
 
             //GETTING OFFSET
-            $offset = $this->getOffset($html_varName);
+            $offset = empty($this->getOffset($html_varName)) ? 0:$this->getOffset($html_varName);
```

Comment:
formatting issue: please change ? 0:$this->getOffset($html_varName); to 

`? 0 : $this->getOffset($html_varName);`